lib/iris/tests/test_analysis.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2010 - 2014, Met Office
+# (C) British Crown Copyright 2010 - 2015, Met Office
```

Comment:
When I didn't include this change it was failing the license test (you can see this in travis build 4063, line 3129). I'm not sure why it wasn't happy with leaving it at 2014 since I did undo the changes I had made.

I've just reset back to a previous state and made the changes again